packages/networks/src.ts/hedera_networks.ts

```diff
@@ -0,0 +1,89 @@
+// @ts-ignore
+import {AccountID} from "@hashgraph/proto";
```

Comment:
The same networks are encapsulated in the `@hashgraph/sdk` module, thus this file will be deleted in the next few commits.